superset-frontend/src/views/CRUD/utils.tsx

```diff
@@ -401,15 +401,17 @@ export const getAlreadyExists = (errors: Record<string, any>[]) =>
     .flat();
 
 export const hasTerminalValidation = (errors: Record<string, any>[]) =>
-  errors.some(
-    error =>
-      !Object.entries(error.extra)
-        .filter(([key, _]) => key !== 'issue_codes')
-        .every(
-          ([_, payload]) =>
-            isNeedsPassword(payload) || isAlreadyExists(payload),
-        ),
-  );
+  errors.some(error => {
+    const not_issues_codes = Object.entries(error.extra).filter(
+      ([key, _]) => key !== 'issue_codes',
+    );
+
+    if (not_issues_codes.length === 0) return true;
+
+    return !not_issues_codes.every(
+      ([_, payload]) => isNeedsPassword(payload) || isAlreadyExists(payload),
```

Comment:
`[, payload]`.
I know this was not done by you, but we might as well clean it up